Capstone/API/APImanager.h

```diff
@@ -0,0 +1,16 @@
+//
+//  APImanager.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/5/22.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface APImanager : NSObject
```

Comment:
nit: We want class names to be in camel case
```suggestion
@interface APIManager : NSObject
```